update-sri.js

```diff
@@ -20,12 +22,8 @@ Promise.all(process.argv.slice(3).map(file =>
     read(file, "utf8")
     .then(body => {
         // Replace size badge url
-        // 1 - url prefix: https://img.badgesize.io/Khan/KaTeX/
-        // 2 - url suffix: /dist/katex.min.js?compression=gzip 
-        const badgeRe = /(https:\/\/img\.badgesize\.io\/Khan\/KaTeX\/v)(?:.+)(\/dist\/katex\.min\.js\?compression=gzip)/g;
-        body = body.replace(badgeRe, (m, pre, post) => {
-            return pre + version + post;
-        });
+        // eslint-disable-next-line max-len
+        body = body.replace(/(https:\/\/img\.badgesize\.io\/Khan\/KaTeX\/v)(?:.+)(\/dist\/katex\.min\.js\?compression=gzip)/g, `$1${version}$2`);
```

Comment:
I didn't realize that you could use `$1`, `$2`, etc. for the capture groups in the replacement string, cool!